products/bigquerydatatransfer/api.yaml

```diff
@@ -84,6 +84,34 @@ objects:
           about the format here:
           https://cloud.google.com/appengine/docs/flexible/python/scheduling-jobs-with-cron-yaml#the_schedule_format
           NOTE: the granularity should be at least 8 hours, or less frequent.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'scheduleOptions'
+        required: false
+        description: |
+          Options customizing the data transfer schedule.
+        properties:
```

Comment:
We've recently started trying to prevent people from being able to write empty blocks, and since this nested object has three optional fields, this would allow that. We can mark the fields as exactly_one_of or at_least_one_of. Do either of those apply here?

Reply:
I would be interested to have this available in Terraform. I understand your point about the empty block. I think the `at_least_one_of` could apply here. If you write the block the intent is to provide one of the options. The three of them have default values so it shouldn't be problematic i.e. `startTime` defaults to `now()`, `endDate` defaults to never and  `disableAutoScheduling` defaults to `false`.

> https://cloud.google.com/bigquery-transfer/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs#TransferConfig.ScheduleOptions

Reply:
This PR looks pretty abandoned, so I'm going to go ahead and close it out. Looks like someone else picked up the feature at https://github.com/GoogleCloudPlatform/magic-modules/pull/3895.